cmd/palomad/commands.go

```diff
@@ -111,6 +114,7 @@ func initRootCmd(
 		txCommand(basicManager),
 		keys.Commands(),
 	)
+	wasmcli.ExtendUnsafeResetAllCmd(rootCmd)
```

Comment:
This clears the `wasm` folder on unsafe-reset-all, too. Not related to the testnet command though

Reply:
@byte-bandit do we want to include this in paloma?

Reply:
Yup This is it.